.vuepress/config.js

```diff
@@ -117,6 +117,7 @@ module.exports = {
             '/reference/features/snapshots',
             '/reference/features/stop_words',
             '/reference/features/synonyms',
+            '/reference/features/telemetry',
```

Comment:
The location of this page isn't super important since we will be restructuring the site during the move to Docusaurus (ETA Q4 2021) but I'm still curious if someone thinks there's a better spot for it :)

Reply:
I think the place you choose is nice.
Or maybe in the dropdown of `Resources`?

Reply:
nitpicking: is telemetry an actual feature? It feels a bit out-of-place to me because it is, at best, a necessary evil for users. As in, we have legitimate reasons for collecting data, but it's not something that directly benefits them in the same way e.g. snapshots do.

I prefer @curquiza's suggestion. Alternatively, perhaps `learn/what_is_meilisearch/`, just above `contact us`?

Reply:
Okay, thanks for the feedback. I will take a look at where other doc sites tend to stick their telemetry files, because all of these options seem to have tradeoffs.

Reply:
Okay, thanks for the feedback. I will take a look at where other doc sites tend to stick their telemetry files. All of these options seem to have tradeoffs, so I'd rather follow convention (if there is one).

Reply:
Ended up deciding to go with @guimachiavelli 's suggestion to place it in `/learn/what_is_meilisearch/`. It's not an ideal long-term resting place, since it's not a key aspect of the product that we want to show off to first-time-users. However, considering that this issue has caused some distrust in the community, I feel that putting it front and center like this is strangely fitting. Transparency forever, I guess ✌🏻